python/src/lsst/gauss2d/fit/pybind11.cc

```diff
@@ -36,7 +36,8 @@
 
 PYBIND11_MODULE(_gauss2d_fit, m) {
     m.doc() = "Gauss2DFit Python bindings";
-    py::module::import("gauss2d");
+    // Is this necessary? Apparently not
+    // py::module::import("lsst.gauss2d");
```

Comment:
The impact of having this can be subtle: if you have any signatures in this package that reference types defined in `lsst.gauss2d`, you'll get better docstrings and conversion behavior will be more likely to be correct if you have this import in place.